test/fake_lang/non_jl_test.jl

```diff
@@ -0,0 +1,50 @@
+using Pkg
+Pkg.activate(".")
+Pkg.instantiate()
+
+struct MyFile
+    file::String
+end
+Base.abspath(file::MyFile) = MyFile(Base.abspath(file.file))
+Base.isabspath(file::MyFile) = Base.isabspath(file.file)
+Base.joinpath(str::String, file::MyFile) = Myfile(Base.joinpath(str, file.file))
+Base.normpath(file::MyFile) = Myfile(Base.normpath(file.file))
+Base.isfile(file::MyFile) = Base.isfile(file.file)
+Base.findfirst(str::String, file::MyFile) = Base.findfirst(str, file.file)
+Base.String(file::MyFile) = file.file
+
+function make_module(file::MyFile)
+    exprs = []
+    for line in eachline(file.file)
+       val, name = split(line, '=')
+       push!(exprs, :(function $(Symbol(name))() $val end))
+    end
+    @show exprs
+    Expr(:toplevel, :(baremodule test
+       $(exprs...)
+    end), :(using .test))
+end
+
+function Base.include(mod::Module, file::MyFile)
+    Core.eval(mod, make_module(file))
+end
+Base.include(file::MyFile) = Base.include(Core.Main, file)
+
+using Revise
+function Revise.parse_source!(mod_exprs_sigs::Revise.ModuleExprsSigs, file::MyFile, mod::Module; kwargs...)
+    ex = make_module(file)
+    @show mod_exprs_sigs, file, mod, kwargs
+    Revise.process_source!(mod_exprs_sigs, ex, file, mod; kwargs...)
+end
+try
+    cp("test.program", "tmp.program")
+    m=MyFile("tmp.program")
+    includet(m)
+    @test test.y() == "2"
+    @test test.x() == "1"
+    mv("new_test.program.old", "tmp.program", force=true)
```

Comment:
```julia
ERROR: LoadError: IOError: open("new_test.program.old", 0, 0): no such file or directory (ENOENT)
Stacktrace:
  [1] uv_error
    @ ./libuv.jl:97 [inlined]
  [2] open(path::String, flags::UInt8, mode::Int64)
    @ Base.Filesystem ./filesystem.jl:105
  [3] open
    @ ./filesystem.jl:97 [inlined]
  [4] sendfile(src::String, dst::String)
    @ Base.Filesystem ./file.jl:992
  [5] cp(src::String, dst::String; force::Bool, follow_symlinks::Bool)
    @ Base.Filesystem ./file.jl:384
  [6] rename(src::String, dst::String; force::Bool)
    @ Base.Filesystem ./file.jl:981
  [7] rename
    @ ./file.jl:977 [inlined]
  [8] #mv#15
    @ ./file.jl:426 [inlined]
  [9] top-level scope
    @ ~/.julia/dev/Revise/test/fake_lang/non_jl_test.jl:45
 [10] include(fname::String)
    @ Base.MainInclude ./client.jl:476
 [11] top-level scope
    @ REPL[4]:1
in expression starting at /home/tim/.julia/dev/Revise/test/fake_lang/non_jl_test.jl:39
```

The source file, `"new_test.program.old"`, doesn't exist. Did you forget to commit a file?

Reply:
Good catch. Can you re-run CI?